.github/workflows/ci.yml

```diff
@@ -12,7 +12,7 @@ on:
 env:
   # Common versions
   GO_VERSION: '1.23'
```

Comment:
We should also remove (or bump) this variable, as we are on Go 1.24 now. But unrelated to this PR.